clients/python/raccoon_client/serde/protobuf_serde.py

```diff
@@ -0,0 +1,21 @@
+from google.protobuf.message import Message
+
+from raccoon_client.serde.serde import Serde
+from raccoon_client.serde.wire import Wire
+
+
+class ProtobufSerde(Serde, Wire):
+    def serialise(self, event: Message):
+        if not isinstance(event, Message):
+            raise ValueError("event should be a protobuf message")
+        return event.SerializeToString()  # the name is a misnomer, returns bytes
+
+    def marshal(self, obj: Message):
+        return obj.SerializeToString()
+
+    def unmarshal(self, marshalled_data: bytes, template: Message):
+        template.ParseFromString(marshalled_data)
```

Comment:
this ParseFromString is also ParseFromBytes?

Reply:
Yes. It can take either String or Bytes.